Add resource on physical information for metadata

kzollove opened the pull request:
Fixes #30 and PASTAplus/web-x#24. 

Provides a resource that explains how to obtain and insert physical information for data resources into metadata.

Linked to from Uploading with static links

kzollove committed:
Add resource on physical information for metadata

kzollove committed:
Link to adding physical info

kzollove committed:
Merge branch 'main' into box

clnsmth commented:
This is a useful resource @kzollove, thanks! A couple things:

1. The only piece of information required for download by the EDI repository, and that will have to be manually specified by a user is a static URL (i.e.  `/eml:eml/../physical/distribution/online/url`). In most cases, the other physical metadata is automatically calculated.
2. The context requiring manual calculation of physical metadata is primarily when submitting data to EDI that will be stored offline (e.g. because it's too big).

This page is nice page but some of it needs to be edited to address the points above as well as the context from which it is referenced.

clnsmth commented on templates/resources/uploading-with-static-data-links.md:
```diff
@@ -71,6 +71,7 @@ To insert a download link via [ezEML](https://ezeml.edirepository.org/eml/):
 The data table will now be associated with the provided download link. Submit to EDI via **Send to EDI** or download the XML document with the **Download EML File** button.
 
 >This method can be replicated in the **Other Entities** tab as well. 
+>If you are not uploading data resources to ezEML, remember to manually [add physical metadata information](/templates/resources/adding-physical-information.md). 
```
Only applies to /eml:eml/../physical/distribution/online/url

clnsmth commented on templates/resources/uploading-with-static-data-links.md:
```diff
@@ -85,6 +86,8 @@ To insert a download link via [EMLassembylline](https://ediorg.github.io/EMLasse
 
 <img class="screen-shot" src="/static/images/link-eal2.png" width="70%">
 
+>If you are using EMLassemblyline with web-hosted data resources, remember to manually [add physical metadata information](/templates/resources/adding-physical-information.md). 
```
Only applies to /eml:eml/../physical/distribution/online/url

clnsmth commented on templates/resources/uploading-with-static-data-links.md:
```diff
@@ -85,6 +86,8 @@ To insert a download link via [EMLassembylline](https://ediorg.github.io/EMLasse
+
 ### Manually editing XML
 
 To insert download links manually:
```
Don't encode XML tags, rather embed within back tactics ``.

clnsmth commented on templates/resources/adding-physical-information.md:
```diff
@@ -0,0 +1,69 @@
+# Adding a resource's physical information
+
+Physical information such as file size, MD5 checksum, and number of rows in a table, are important pieces of metadata. When a resource is uploaded to ezEML or processed by EMLassemblyline, this information is automatically calculated. However, neither application can obtain this information from a file that is hosted externally. The responsibility is instead placed on the data provider to determi ...
```
Why are these important pieces of metadata?

kzollove committed:
Resolve suggestion more emphasis on why

Resolve suggestion clarify EAL statement

Resolve suggestion dont encode XML use backticks

Resolve suggestion clarify ezEML statement
